.github/workflows/mititools.yml

```diff
@@ -19,11 +18,8 @@ jobs:
       - run: |
           python -m pip install git+https://github.com/miti-consortium/mititools.git
```

Comment:
It may be cleaner to install the local copy by specifying the path (e.g., `python -m pip install .`)

Reply:
Alas, MITI tools is a different repo.
But for local repositories, completely agree!